public/css/main.css

```diff
@@ -10,11 +10,21 @@
   font-style: normal;
 }
 
+html {
+  height: 100%;
+}
+
 body {
   background-image: radial-gradient(circle, #3241a6 0, #202a6b 120%);
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI",
     "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue",
     "Arial", sans-serif;
+  height: 100%;
+  min-height: 100vh;
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+  display: flex;
+  flex-direction: column;
```

Comment:
Can we use utility classes for some or all of this? https://github.com/tachyons-css/tachyons-flexbox